teps/0119-add-taskrun-template-in-pipelinerun.md

```diff
@@ -0,0 +1,179 @@
+---
+status: proposed
+title: Add taskRun template in PipelineRun
+creation-date: '2022-08-12'
+last-updated: '2022-08-12'
+authors:
+- '@yuzp1996'
+---
+
+# TEP-0119: Add taskRun template in PipelineRun
+
+<!-- toc -->
+- [Summary](#summary)
+  - [Use Cases](#use-cases)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+  - [taskRunTemplate](#implement)
+  - [Test Plan](#test-plan)
+<!-- /toc -->
+
+## Summary
+It is now supported for users to specify spec for a specific taskRun when creating a pipelineRun, but it is not supported for users to set spec for all taskRun.
+
+If you want to specify common configurations for all taskRun, you must specify them individually.
+
+This Tep attempts to provide a convenient way to specify the run specification of all tasks in pipelineRun.
+
+### Use Cases
+Pipeline and Task User:
+
+I would like to use the same service account for all of my pipeline tasks except one.
+
+I want to specify compute resources that each TaskRun in my PipelineRun should run with, and don't want to have to specify them individually for each TaskRun.
+
+## Proposal
+Add filed taskRunTemplate to PipelineRun.Spec then users can specify common configuration in taskRunTemplate and the configuration will apply to all the TaskRun. 
```

Comment:
```suggestion
Add field taskRunTemplate to PipelineRun.Spec so that users can specify common configuration in taskRunTemplate that will apply to all the TaskRuns. 
```